### ISSUE TITLE
[Security Advisory] Dependency has a vulnerability

### ISSUE THREAD

**Issue by ProtonScott** [symfony/webpack-encore#611]
An npm audit triggered this notification when I updated today:

```
  High            Prototype Pollution

  Package         set-value

  Patched in      >=3.0.1

  Dependency of   @symfony/webpack-encore [dev]

  Path            @symfony/webpack-encore > webpack > watchpack > chokidar >
                  anymatch > micromatch > snapdragon > base > cache-base >
                  set-value

  More info       https://npmjs.com/advisories/1012
```

[You can check the advisory here](https://npmjs.com/advisories/1012)

In accordance to this, it appears this issue has already been patched.

When I went looking at watchpack/webpack github repositories, I did not find this dependency among their package.json files. I think the cause here is an old version of webpack that encore forces.

Can we get a proper update for this pushed out?

Its causing havoc for those of us who are beholden to our bosses who only see the audit output in the CI with no other context 😆

**Comment by Kocal** [comment]
Maybe it would be nice to enable automatic GitHub security alerts for dependencies vulnerabilities. 🤔

**Comment by Lyrkan** [comment]
Hi @ProtonScott,

Sadly I don't think that we can do anything from our side since we already allow people to get the latest stable (4.35.3) version of Webpack:

* `webpack@4.35.3` requires `micromatch@^3.1.8` which gets resolved to `micromatch@3.1.10`
* `micromatch@3.1.10` requires `snapdragon@^0.8.1` which gets resolved to `snapdragon@0.8.2`
* `snapdragon@0.8.2` requires `base@^0.11.1` which gets resolved to `base@0.11.2`
* `base@0.11.2` requires `cache-base@^1.0.1` which gets resolved to `cache-base@1.0.1`
* `cache-base@1.0.1` requires `set-value@^2.0.0` which gets resolved to `set-value@2.0.1`

@Kocal We do have security alerts enabled on the repository, but none was triggered for that issue (yet).

**Comment by Lyrkan** [comment]
Closing this since it seems that it is actually a false positive and no update is needed if you already have `set-value@2.0.1` (see https://www.npmjs.com/advisories/1012/versions).
